includes/class-jwt-auth-schema-filters.php

```diff
@@ -16,14 +16,90 @@
  * Class JWT_Auth_Schema_Filters
  */
 class JWT_Auth_Schema_Filters {
+	/**
+	 * Adds filters.
+	 *
+	 * @return string|null
+	 */
+	public static function get_auth_class() {
+		if ( class_exists( '\WPGraphQL\JWT_Authentication\Auth' ) ) {
+			return '\WPGraphQL\JWT_Authentication\Auth';
+		} elseif ( class_exists( \WPGraphQL\Login\Main::class ) ) {
+			return \WPGraphQL\Login\Auth\TokenManager::class;
+		} else {
+			return null;
+		}
+	}
+
+	/**
+	 * Get the auth token for a user.
+	 *
+	 * @param \WP_User $user The user object.
+	 * 
+	 * @throws \GraphQL\Error\UserError If the token cannot be retrieved.
+	 *
+	 * @return string|null
+	 */
+	public static function get_auth_token( \WP_User $user ) {
+		$auth_class = self::get_auth_class();
+
+		if ( ! $auth_class ) {
+			return null;
+		}
+
+		/**
+		 * This method is typed wrong upstream.
+		 *
+		 * @var \WP_Error|string|null $token
+		 */
+		$token = $auth_class::get_token( $user );
```

Comment:
It's [`WPGraphQL\Login\Auth\TokenManager::get_auth_token()`](https://github.com/kidunot89/wp-graphql-headless-login/blob/1911066a561afda2f37e183d5730ed166de4508d/src/Auth/TokenManager.php#L69).

Can probably use a `method_exists( $auth_class, ...)` to call the correct method.